src/utils.js

```diff
@@ -223,6 +223,9 @@ export function mergeObjects(obj1, obj2, concatArrays = false) {
 }
 
 export function asNumber(value) {
+  if(value === "") {
```

Comment:
Please add a space after `if`. Would be nice to add an eslint rule for this if you have the time btw :)